indico/modules/attachments/util.py

```diff
@@ -104,8 +104,6 @@ def can_manage_attachments(obj, user):
         return True
     if isinstance(obj, db.m.Event) and obj.can_manage(user, 'submit'):
         return True
-    if isinstance(obj, db.m.Session) and obj.can_manage(user, 'coordinate'):
```

Comment:
I don't remember what exactly but there was a usecase for it so let's not remove it for now.